spec/views/sign_up/recovery_codes/show.html.slim_spec.rb

```diff
@@ -37,6 +37,8 @@
   end
 
   it 'displays the correct progress step' do
```

Comment:
maybe update the spec description to be "when @show_progress_bar is true" or something?